acceptance/acceptance_test.go

```diff
@@ -300,6 +300,10 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 				items := strings.Split(stub.Pattern, " ")
 				require.Len(t, items, 2)
 				server.Handle(items[0], items[1], func(req testserver.Request) any {
+					if stub.DelaySeconds > 0 {
+						time.Sleep(time.Duration(stub.DelaySeconds) * time.Second)
```

Comment:
Does this work for fractional DelaySeconds? e.g. DelaySeconds=0.001. It seems it first type casted to integer, losing info.

Reply:
Good catch, fixed the type casting.